### PR TITLE
fix(curriculum): changed test text to use should for Front End Libraries

### PR DESCRIPTION
**Pull request by RandellDawson**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Related to PR #36860 

This PR makes sure all challenges' tests in the `Front End Libraries` section use "should " or "should not".

### CODE REVIEW

File: curriculum/challenges/english/03-front-end-libraries/bootstrap/call-out-optional-actions-with-btn-info.english.md [hunk 1]
```diff
@@ -22,13 +22,13 @@ Note that these buttons still need the <code>btn</code> and <code>btn-block</cod
...
 tests:
-  - text: Create a new <code>button</code> element with the text "Info".
+  - text: You should create a new <code>button</code> element with the text "Info".
     testString: assert(new RegExp("info","gi").test($("button").text()));
   - text: Both of your Bootstrap buttons should have the <code>btn</code> and <code>btn-block</code> classes.
     testString: assert($("button.btn-block.btn").length > 1);
   - text: Your new button should have the class <code>btn-info</code>.
     testString: assert($("button").hasClass("btn-info"));
-  - text: Make sure all your <code>button</code> elements have a closing tag.
+  - text: All your <code>button</code> elements should have closing tags.
```
Comment (moT01) [inline curriculum/challenges/english/03-front-end-libraries/bootstrap/call-out-optional-actions-with-btn-info.english.md]:
```suggestion
  - text: All of your <code>button</code> elements should have closing tags.
```

File: curriculum/challenges/english/03-front-end-libraries/bootstrap/create-a-bootstrap-headline.english.md [hunk 2]
```diff
@@ -23,9 +23,9 @@ Color your <code>h3</code> element with the <code>text-primary</code> Bootstrap
 
 ```yml
 tests:
-  - text: Add a <code>h3</code> element to your page.
+  - text: You should add a <code>h3</code> element to your page.
```
Comment (moT01) [inline curriculum/challenges/english/03-front-end-libraries/bootstrap/create-a-bootstrap-headline.english.md]:
```suggestion
  - text: You should add an <code>h3</code> element to your page.
```

File: curriculum/challenges/english/03-front-end-libraries/jquery/delete-your-jquery-functions.english.md [hunk 3]
```diff
@@ -23,15 +23,15 @@ Delete all three of these jQuery functions from your <code>document ready functi
...
 ```yml
 tests:
-  - text: Delete all three of your jQuery functions from your <code>document ready function</code>.
+  - text: All three of your jQuery functions should be deleted from your <code>document ready function</code>.
     testString: assert(code.match(/\{\s*\}\);/g));
-  - text: Leave your <code>script</code> element intact.
+  - text: You should leave your <code>script</code> element intact.
     testString: assert(code.match(/<script>/g));
-  - text: Leave your <code>$&#40document&#41.ready&#40function&#40&#41 {</code> to the beginning of your <code>script</code> element.
+  - text: YOu should leave your <code>$&#40document&#41.ready&#40function&#40&#41 {</code> to the beginning of your <code>script</code> element.
```
Comment (moT01) [inline curriculum/challenges/english/03-front-end-libraries/jquery/delete-your-jquery-functions.english.md]:
```suggestion
  - text: You should leave your <code>$&#40document&#41.ready&#40function&#40&#41 {</code> at the beginning of your <code>script</code> element.
```

File: curriculum/challenges/english/03-front-end-libraries/jquery/target-the-children-of-an-element-using-jquery.english.md [hunk 4]
```diff
@@ -27,7 +27,7 @@ tests:
     testString: assert($("#right-well").children().css("color") === 'rgb(255, 165, 0)');
   - text: You should use the <code>children&#40&#41</code> function to modify these elements.
     testString: assert(code.match(/\.children\(\)\.css/g));
-  - text: Only use jQuery to add these classes to the element.
+  - text: You should onlu use jQuery to add these classes to the element.
```
Comment (moT01) [inline curriculum/challenges/english/03-front-end-libraries/jquery/target-the-children-of-an-element-using-jquery.english.md]:
```suggestion
  - text: You should only use jQuery to add these classes to the element.
```

**Review by moT01 — CHANGES_REQUESTED** [review]

**Comment by RandellDawson** [comment]
@moT01 I have committed your suggested changes.

**Review by moT01 — APPROVED** [review]
🎉 LGTM 🎉

**Review by cmccormack — APPROVED** [review]
These all look good to me; however, I would have preferred to have `you` and `your` removed from most of the comments as it seems a bit personal.

For example:
```yaml
- text: You should add an <code>i</code> element with the classes <code>fas</code> and <code>fa-thumbs-up</code>.
```
could be rewritten
```yaml
- text: An <code>i</code> element with the classes <code>fas</code> and <code>fa-thumbs-up</code> should be added.
```

Again, this looks good to me and I approve as-is, if you agree with what I stated above I could do the work to go through them again, just let me know.

**Review by Manish-Giri — APPROVED** [review]
LGTM 💯

**Comment by RandellDawson** [comment]
@cmccormack According to the current [Writing challenge descriptions/instructions](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/docs/how-to-work-on-coding-challenges.md#writing-challenge-descriptionsinstructions) section, the challenge text should be written using the second person "you" and most of the tests in the curriculum seem to be written this way.  I do not think it necessary, but you can create a separate issue where we can all discuss it I suppose.  In the mean time, I think this PR can be merged now.

**Comment by ojeytonwilliams** [comment]
I agree that there's no need to hold this up pending a change in the style guide.

**Pull request merged**